wiring/inc/spark_wiring_ble.h

```diff
@@ -927,6 +928,26 @@ class BleLocalDevice {
     int8_t txPower() const;
     int selectAntenna(BleAntennaType antenna) const;
 
+    // Access provisioning mode
+    int provisioningMode(bool enabled) const;
+    bool getProvisioningStatus() const;
+
+    template<typename T1, typename T2, typename T3>
+    int setProvisioningUuids(T1 svcUuid, T2 txUuid, T3 rxUuid) const {
+        BleUuid tempSvcUUID(svcUuid);
+        BleUuid tempTxUUID(txUuid);
+        BleUuid tempRxUUID(rxUuid);
+        return system_set_prov_svc_uuid(tempSvcUUID.halUUID(), tempTxUUID.halUUID(), tempRxUUID.halUUID(), nullptr);
+    }
+
+    template<typename T>
+    int setProvisioningAdvServiceUuid(T uuid) const {
```

Comment:
We don't need this API. System layer can fetch the provisioning service UUID from the BLE control request class.